test/AbstractQuestionAssertion.php

```diff
@@ -0,0 +1,52 @@
+<?php
+declare(strict_types=1);
+
+namespace LaminasTest\ComponentInstaller;
+
+use function strpos;
+
+/**
+ * @psalm-immutable
```

Comment:
This is an abstract class... is it really immutable?

Reply:
Yes, because I dont want to have it changed from the outside. This avoids misusage when extending this class.